src/modules/constants/syncRichText.ts

```diff
@@ -0,0 +1,3 @@
+export const customAssetCodenameAttributeName = "data-asset-codename";
```

Comment:
Add a comment explaining that these attributes are not supported by the MAPI. That we only use those to store the codename for syncing in the file format and are never sent to MAPI.